test/unit/rules/no-multiple-empty-lines-test.js

```diff
@@ -76,6 +80,7 @@ generateRuleTests({
       config: { max: 3 },
 
       template: '<div>foo</div>\n\n\n\n\n<div>bar</div>',
+      fixedTemplate: '<div>foo</div>\n\n\n\n<div>bar</div>',
```

Comment:
Shouldn't this be collapsed to `\n\n` as well? Always collapse to one blank line, per my other comment.

Reply:
max is 3 here! so 3 lines as per config!

Reply:
Missed that, thanks!